### ISSUE TITLE
table formatting doing the wrong thing

### ISSUE THREAD

**Issue by despairblue** [zhuochun/md-writer#59]
```
| Header One     | Header Two     |
| :------------- | :------------- |
| Item One       | Item Two       |
```
is formatted into
```
 | Header One | Header Two |
-|------------|------------|-
 | Item One   | Item Two   |
```

Which adds another column at the beginning and the end.

**Comment by zhuochun** [comment]
Thanks for the bug report :+1: It will be fixed in the next release

**Comment by despairblue** [comment]
:clap: thx

**Comment by zhuochun** [comment]
Fixed in the v1.8.1 Please try again :package:

**Comment by despairblue** [comment]
Oh, sry I was busy :sweat:   

Yes it works, thank you :+1:
